tests/integration.rs

```diff
@@ -251,7 +256,8 @@ fn golden_reference(args: &[&str], file: &str) -> TestResult {
     if std::env::var("BLESS").as_deref() == Ok("1") {
         fs::write(file, actual)?;
     } else {
-        let expected = fs::read_to_string(file).context("failed to read usage file")?;
+        let mut expected = fs::read_to_string(file).context("failed to read usage file")?;
+        content_modifier(&mut expected);
```

Comment:
nit: please call `content_normalize` directly here, I don't think it needs to be customizable per-test.

Reply:
Fixed in [`355afec` (#79)](https://github.com/holmgr/cargo-sweep/pull/79/commits/355afecc621a96cefcd3c6c97dd71d26c8c5effc)